Google.Solutions.Compute/Extensions/GetSerialPortOutputStream.cs

```diff
@@ -0,0 +1,117 @@
+﻿//
+// Copyright 2019 Google LLC
+//
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+// 
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+using Google.Apis.Compute.v1;
+using System;
+using System.Threading.Tasks;
+
+namespace Google.Solutions.Compute.Extensions
+{
+    public static class GetSerialPortOutputStreamExtensions
+    {
+        public static SerialPortStream GetSerialPortOutputStream(
```

Comment:
note, this isn't in the currernt release, yea?  we're just first going to support rdp andnto the serial or reset password?

Reply:
This does not have anything to do with RDP or IAP tunneling, really, but it is needed for `ResetWindowsUser`.